validator/validator-main.protoascii

```diff
@@ -5321,6 +5320,39 @@ tags: {
   }
 }
 
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: 'I-AMPHTML-INTRINSIC-SIZER'
+  mandatory_parent: "I-AMPHTML-SIZER"
+  attrs: {
+    name: "class"
+    value_regex: "i-amphtml-intrinsic-sizer"
+    mandatory: true
```

Comment:
Assuming the above, I would add to this attrs spec:

dispatch_key: NAME_VALUE_REGEX

Reply:
It will force the validator to use this tagspec for matching if and only if the IMG tag has a class="i-amphtml-intrinsic-sizer" attribute and value. This is useful in terms of improving error message specificity.

Reply:
Seems like NAME_VALUE_REGEX is not defined.

Reply:
Oops. NAME_VALUE_DISPATCH.